### PR TITLE
Make network cookbook compatible with Leap

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: chef/cookbooks/network/attributes/default.rb [hunk 1]
```diff
@@ -18,8 +18,11 @@
   default[:network][:base_pkgs] = ["bridge-utils",
                                    "vlan"]
   default[:network][:ovs_pkgs] = ["openvswitch",
```
Comment (rsalevsky) [inline chef/cookbooks/network/attributes/default.rb]: Makes it sense to remove the line break? The intention is anyway wrong so I prefer removing the line break or
```ruby
default[:network][:ovs_pkgs] = [
  "openvswitch",
  "openvswitch-switch"
]
```

Comment (vuntz) [inline chef/cookbooks/network/attributes/default.rb]: @rsalevsky added a commit for this

**Comment by rsalevsky** [comment]
+1

**Comment by aplanas** [comment]
I was confused because the link : )

+1

File: chef/cookbooks/network/attributes/default.rb [hunk 2]
```diff
@@ -15,11 +15,18 @@
...
+  default[:network][:ovs_pkgs] = [
+    "openvswitch",
+    "openvswitch-switch"
+  ]
+  # openSUSE uses the module shipped with upstream kernel
+  if node[:platform] == "suse"
+    default[:network][:ovs_pkgs].push("openvswitch-kmp-default")
+  end
   # SLES11 uses a different service name for openvswitch
   if node[:platform] == "suse" && node[:platform_version].to_f < 12.0
```
Comment (tboerger) [inline chef/cookbooks/network/attributes/default.rb]: I think we can drop the SLE 11 features?

File: chef/cookbooks/network/attributes/default.rb [hunk 3]
```diff
@@ -27,18 +34,26 @@
...
-                                   "vlan"]
-  default[:network][:ovs_pkgs] = ["linux-headers-#{`uname -r`.strip}",
-                                  "openvswitch-datapath-dkms",
-                                  "openvswitch-switch"]
+  default[:network][:base_pkgs] = [
+    "bridge-utils",
+    "vlan"
+  ]
+  default[:network][:ovs_pkgs] = [
+    "linux-headers-#{`uname -r`.strip}",
```
Comment (tboerger) [inline chef/cookbooks/network/attributes/default.rb]: To be really correct I would replace this with `"linux-headers-#{node["kernel"]["release"]}",`

**Comment by tboerger** [comment]
Small NITPIK but otherwise :+1:

File: chef/cookbooks/network/attributes/default.rb (hunk 2 above)
Comment (dirkmueller) [inline chef/cookbooks/network/attributes/default.rb]: Unrelated to the purpose of this PR, cleanup can be done later

**Comment by dirkmueller** [comment]
added "do not merge" until M8 is released.

**Comment by aplanas** [comment]
Time to merge?

**Pull request merged**